components/app.js

```diff
@@ -40,5 +44,32 @@ class App {
 
   start() {
     this.getGrades();
+    this.gradeForm.onSubmit(this.createGrade);
+  }
+
+  createGrade(name, course, grade) {
+    console.log("Test: createGrade method", name, course, grade);
+    $.ajax({
+      method: "POST",
+      url: "https://sgt.lfzprototypes.com/api/grades",
+      headers: {
+        "X-Access-Token": "R1JRk9us",
+      },
+      data: {
+        "name": `${name}`,
```

Comment:
These template literals are not necessary, you can just input the variable directly

Reply:
okay.. I modified those codes.